examples/app-playwright/package.json

```diff
@@ -16,7 +16,7 @@
     "vue-router": "^4.2.5"
   },
   "devDependencies": {
-    "playwright": "^1.42.1",
-    "@nuxt/test-utils": "latest"
+    "@nuxt/test-utils": "latest",
+    "@playwright/test": "1.43.0-alpha-2024-03-14"
```

Comment:
Note: thats the alpha version with the fix:  https://github.com/microsoft/playwright/pull/29865